src/icons/general/IconChat.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+
+import AccessibleSVG from '../accessible-svg';
+
+import { Icon } from '../iconTypes';
+
+const IconChat = ({ className = '', color = '#979797', height = 16, title, width = 16 }: Icon) => (
```

Comment:
Same comment as the other. Won't do each one but if we want to clean that up, it would be good to move in one direction, or ignore it.